pkg/exec/schema/exec.json

```diff
@@ -1,84 +1,60 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Is `draft-07` still the version that these schemas represent?

Reply:
I was working off of the current json schema draft (07) which has been out for years when I wrote these schemas. So I could try to diff the drafts and figure out if they are actually compatible with older drafts but that's a ton of work. 

I'm inclined to chalk up the problems so far to bugs, not older draft versions until I get clear answers on what version of the draft the libs used by VS Code is actually using.